### PR TITLE
BBE Landing: Add localized images for slideshow

### PR DESCRIPTION
**Pull request by aneeshd16**
#### Proposed Changes

P2: pdh1Xd-1DL-p2

This PR updates the images used on the BBE landing page. Changes:
* Show a separate set of localized images if the locale is `es`.
* The earlier implementation had hardcoded values for the animation. This change refactors the component so that the animation is easier to understand, and any future changes to the images would not involve any changes to the animation.
* The number of files changed in this PR is large, so it might be easier to review the change one commit at a time.
* The files `theme-grid.png, vertical-creative.png, vertical-restaurant.png and vertical-services.png` are not being used, so I deleted those too.

#### Testing Instructions

* Switch to EN locale.
* Go to `/setup/site-setup/goals?siteSlug=<site slug>`.
* Confirm that the slideshow displays and animates correctly. You can verify that the animation is correct by comparing it to the live version (`https://wordpress.com/setup/site-setup/difmStartingPoint?siteSlug=<site slug>`)
* Switch to the ES locale and repeat the above steps.
* Confirm that localized images are shown.

#### Pre-merge Checklist

- [x] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [x] Have you checked for TypeScript, React or other console errors?
- [x] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

Fixes Automattic/martech#1329

### CODE REVIEW

**Comment by aneeshd16** [comment]
This has been tested by @mikeicode. (pdh1Xd-1DL-p2#comment-1880)

File: client/my-sites/marketing/do-it-for-me/site-build-showcase.tsx [hunk 1]
```diff
@@ -1,60 +1,125 @@
...
-		<Container>
-			<div style={ { backgroundImage: `url( ${ site1 } )` } } />
-			<div style={ { backgroundImage: `url( ${ site2 } )` } } />
-			<div style={ { backgroundImage: `url( ${ site3 } )` } } />
-			<div style={ { backgroundImage: `url( ${ site4 } )` } } />
-			<div style={ { backgroundImage: `url( ${ site5 } )` } } />
-		</Container>
+		<AnimatedSlideshow slideCount={ images.length } interval={ 8 } fadeDelay={ 2.5 }>
+			{ images.map( ( image, index ) => (
+				<div key={ '' + index } style={ { backgroundImage: `url( ${ image } )` } } />
```
Comment (southp) [inline client/my-sites/marketing/do-it-for-me/site-build-showcase.tsx]: Could you help me understand the difference between `{ '' + index }` and `{ index }` here? I feel there could be nuances I'm missing here :)

**Review by southp — APPROVED** [review]
I've also confirmed it works as the test plan explained. Beautiful refactor to the animation code 💯 There is a inline comment regarding an expression that I'm not sure about, but overall it LGTM.

File: client/my-sites/marketing/do-it-for-me/site-build-showcase.tsx (hunk 1 above)
Comment (aneeshd16) [reply client/my-sites/marketing/do-it-for-me/site-build-showcase.tsx]: Nice catch! I had this faint notion that the `key` prop must be a string, but I cannot find any references to support that. I'll change this back to `{ index }`.

**Pull request merged**